database/schema.js

```diff
@@ -22,16 +38,9 @@ const productSchema = new mongoose.Schema({
 
 const Product = mongoose.model('Product', productSchema);
 
-const firstTestProduct = new Product({
-  productNumber: 1,
-  associatedProductNumbers: [2, 3, 4, 5, 6]
-})
+db.saveCollection = async (incomingEntries) => {
+  await Product.create(incomingEntries)
+  db.close();
```

Comment:
@masoncpott This is cool. So you don't get "leaky" test error on Jest. When I did mine in MySQL, I couldn't close the connection after each query and got warnings.